aries-site/src/layouts/main/Layout.js

```diff
@@ -143,6 +193,41 @@ export const Layout = ({
                 </Box>
               </Main>
             </MainContentWrapper>
+            <FeedbackButton
+              margin={{ vertical: 'large', horizontal: 'medium' }}
```

Comment:
```suggestion
              margin={{ vertical: 'medium', horizontal: 'medium' }}
```
To me it looks a little awkward having the horizontal and vertical margins as different, just my opinion:

<img width="1662" alt="Screen Shot 2022-10-03 at 12 06 30 PM" src="https://user-images.githubusercontent.com/12522275/193658984-b9611645-b30b-42a8-98bf-d55d8c0d7269.png">